src/pybind/mgr/dashboard_v2/services/ceph_service.pyi

```diff
@@ -0,0 +1,5 @@
+from ..module import Module
```

Comment:
I would remove this file. Although I understand the reason why you are creating it, it feels strange that an interface only declares one attribute, and does not also declare all methods that are part of `CephServiceMixin` implementation. Instead of creating this file, I would add a class comment to the `CephServiceMixin` explaining that this mixin should be used with controller classes.

But if you decide to still stick with this file, could you please add above the first line:

```
# -*- coding: utf-8 -*-
from __future__ import absolute_import
```

Reply:
As this is the best way I found to make IDEs aware of the type of the `mgr` attribute within the `CephServiceMixin`, I'd love to keep that file. It's not meant to be used as a complete interface to `CephServiceMixin`.

Reply:
I continue to not like to have this file. And with the latest additions I'm even more against it. This will be a maintenance burden, and someone will eventually forget to add the necessary info here.